src/test/java/foodwhere/logic/commands/SDeleteCommandTest.java

```diff
@@ -17,47 +17,47 @@
 
 /**
  * Contains integration tests (interaction with the Model) and unit tests for
- * {@code DeleteCommand}.
+ * {@code SDeleteCommand}.
  */
-public class DeleteCommandTest {
+public class SDeleteCommandTest {
 
     private Model model = new ModelManager(TypicalStalls.getTypicalAddressBook(), new UserPrefs());
 
     @Test
     public void execute_validIndexUnfilteredList_success() {
         Stall stallToDelete = model.getFilteredStallList().get(TypicalIndexes.INDEX_FIRST_STALL.getZeroBased());
-        DeleteCommand deleteCommand = new DeleteCommand(TypicalIndexes.INDEX_FIRST_STALL);
+        SDeleteCommand SDeleteCommand = new SDeleteCommand(TypicalIndexes.INDEX_FIRST_STALL);
 
-        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_STALL_SUCCESS, stallToDelete);
+        String expectedMessage = String.format(SDeleteCommand.MESSAGE_DELETE_STALL_SUCCESS, stallToDelete);
 
         ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
         expectedModel.deleteStall(stallToDelete);
 
-        assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
+        assertCommandSuccess(SDeleteCommand, model, expectedMessage, expectedModel);
     }
 
     @Test
     public void execute_invalidIndexUnfilteredList_throwsCommandException() {
         Index outOfBoundIndex = Index.fromOneBased(model.getFilteredStallList().size() + 1);
-        DeleteCommand deleteCommand = new DeleteCommand(outOfBoundIndex);
+        SDeleteCommand SDeleteCommand = new SDeleteCommand(outOfBoundIndex);
```

Comment:
This may cause checkstyle issues as the variable name is not following camelCase